internal/util/ssa/filterintent.go

```diff
@@ -35,10 +35,10 @@ type FilterObjectInput struct {
 	// NOTE: ignore paths which point to an array are not supported by the current implementation.
 	IgnorePaths []contract.Path
 
-	// DropEmptyStruct instruct the Helper to drop all fields with values equals to empty struct.
+	// DropEmptyStructAndNil instruct the Helper to drop all fields with values equals to empty struct.
```

Comment:
```suggestion
	// DropEmptyStructAndNil instruct the Helper to drop all fields with values equals to empty struct or nil.
```
Q: why this was necessary? (when we will have non relevant nil in the yaml?)

Reply:
metav1.Time becomes nil even with omitzero